server/handlers/post_account.go

```diff
@@ -3,21 +3,18 @@ package handlers
 import (
 	"net/http"
 
-	"github.com/keratin/authn-server/server/sessions"
 	"github.com/keratin/authn-server/app"
-	"github.com/keratin/authn-server/lib/route"
 	"github.com/keratin/authn-server/app/services"
+	"github.com/keratin/authn-server/lib/route"
+	"github.com/keratin/authn-server/server/sessions"
 )
 
 func PostAccount(app *app.App) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		// Create the account
 		account, err := services.AccountCreator(
 			app.AccountStore,
-			app.Config,
-			r.FormValue("username"),
-			r.FormValue("password"),
-		)
+			app.Config, r.FormValue("username"), r.FormValue("password"), r.FormValue("name"), "")
```

Comment:
add picture here also.. could be added if we upload a picture before posting a new account.....